core/src/main/java/com/linecorp/armeria/internal/client/DefaultClientRequestContext.java

```diff
@@ -383,6 +390,29 @@ public CompletableFuture<Boolean> init() {
         }
     }
 
+    @Override
+    public boolean initAndFail(Throwable cause) {
+        if (!initializationTriggeredUpdater.compareAndSet(this, 0, 1)) {
+            return false;
+        }
+        acquireEventLoop(endpointGroup);
```

Comment:
Probably, we don't need to call `initializeResponseCancellationScheduler` in `acquireEventLoop`?